packages/hawtio/package.json

```diff
@@ -45,13 +45,14 @@
     "@types/dagre": "^0.7.50",
     "@types/dagre-layout": "^0.8.3",
     "@types/jest": "^29.5.5",
+    "@types/jquery": "^3.5.19",
     "@types/node": "^18.17.18",
     "@types/react": "^18.2.21",
     "@types/react-dom": "^18.2.7",
     "@types/react-router-dom": "^5.3.3",
     "dagre": "^0.8.5",
     "eventemitter3": "^5.0.1",
-    "jolokia.js": "^1.7.3-2",
+    "jolokia.js": "^2.0.0-dev.9",
```

Comment:
Is a dev release a viable version we should be depending on? Presumably, this soak-tests it before doing a full release of both `jolokia` and `hawtio-next` ?

Reply:
Good point. What's changed really between `1.7.3` and `2.0.0-dev.x` is the type definitions only and the main `jolokia.js` and `jolokia-simple.js` haven't been changed. So I can safely say we can depend on it even for the current Jolokia 1.7.x agent. After all, who wrote the type definitions is me and I know it works ;-)